example/spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -99,7 +99,7 @@ function isExecutionForGranuleId(taskInput, params) {
 describe('The S3 Ingest Granules workflow', () => {
   const inputPayloadFilename = './spec/parallel/ingestGranule/IngestGranule.input.payload.json';
   const providersDir = './data/providers/s3/';
-  const collectionsDir = './data/collections/s3_MOD09GQ_006';
+  const collectionsDir = './data/collections/s3_MOD09GQ_006_full_ingest';
```

Comment:
To avoid having tests by default backup files to LZARDS as part of the collection configuration (presuming they're using the IngestAndPublish workflow).    This minimizes the failure footprint if their service goes down/is being updated/etc while still allowing the lambda to run (without attempting to run backups) in workflows where the config is not defined.